src/App/Pages/Settings/SecuritySettingsPageViewModel.cs

```diff
@@ -467,6 +479,15 @@ private void TriggerUpdateCustomVaultTimeoutPicker()
             TriggerPropertyChanged(nameof(CustomVaultTimeoutTime));
         }
 
+        private void TriggerVaultTimeoutActionLockAllowedPropertyChanged()
+        {
+            MainThread.BeginInvokeOnMainThread(() =>
+            {
+                TriggerPropertyChanged(nameof(IsVaultTimeoutActionLockAllowed));
+                TriggerPropertyChanged(nameof(SetUpUnlockMethodLabel));
+            });
```

Comment:
🎨 Add a call to `VaultTimeoutActionPickerViewModel.SelectOptionCommand.RaiseCanExecuteChanged();` so the command's binded controls are updated given the `canExecute` changed.